packages/notifi-react/lib/context/GlobalStateContext.tsx

```diff
@@ -18,7 +18,13 @@ export type GlobalError = {
   errorData?: string; // Allowing to pass custom string data to be parsed by JSON.parse
 };
 
-type GlobalCtas = Record<string, () => void>;
+export enum CtaTypeEnum {
+  onClose = 'onClose',
+}
+
+export type CtaType = `${CtaTypeEnum}`;
+
+export type GlobalCtas = Record<CtaType, () => void>;
```

Comment:
If the value is always `()=> void`, why don't we just simply set an Unioin type for Cta like below
```ts
export type Cta = 'onClose';

export type GlobalCtas = Record<Cta, () => void>;
```
This also reserves the flexibility if in the future like below:
```ts
export type Cta = 'onClose' | 'onXXX'

export type GlobalCtas = Record<Cta, Cta extends 'onClose' ? () => void : (sth: string) => void>;
```